generated/nidmm/session.py

```diff
@@ -2852,7 +2853,7 @@ def read_multi_point(self, maximum_time, array_size):
         actual_number_of_points_ctype = ctypes_types.ViInt32_ctype(0)
         error_code = self.library.niDMM_ReadMultiPoint(self.vi, maximum_time, array_size, ctypes.cast(reading_array_ctype, ctypes.POINTER(ctypes_types.ViReal64_ctype)), ctypes.pointer(actual_number_of_points_ctype))
         errors._handle_error(self, error_code)
-        return [reading_array_ctype[i].value for i in range(array_size)], actual_number_of_points_ctype.value
+        return [python_types.ViReal64(reading_array_ctype[i].value) for i in range(array_size)], python_types.ViInt32(actual_number_of_points_ctype.value)
```

Comment:
If we determine at some point that we need to address performance concerns with creating this list; and want to return the ctypes object directly because it has the right interface, or want to return a numtypes object because it's more performant... We most likely won't be able to return ViReal64s anymore. Is that ok?